src/postamble_modularize.js

```diff
@@ -18,7 +18,7 @@ moduleRtn = Module;
 
 #endif // WASM_ASYNC_COMPILATION
 
-#if ASSERTIONS
+#if ASSERTIONS && MODULARIZE != 'instance'
```

Comment:
Why this change?

Reply:
Not needed, this was from before the init function wrapped everything.